sdk/confluent/Microsoft.Azure.Management.Confluent/src/Generated/Models/UserDetail.cs

```diff
@@ -30,10 +30,10 @@ public UserDetail()
         /// <summary>
         /// Initializes a new instance of the UserDetail class.
         /// </summary>
+        /// <param name="emailAddress">Email address</param>
         /// <param name="firstName">First name</param>
         /// <param name="lastName">Last name</param>
-        /// <param name="emailAddress">Email address</param>
-        public UserDetail(string firstName = default(string), string lastName = default(string), string emailAddress = default(string))
+        public UserDetail(string emailAddress, string firstName = default(string), string lastName = default(string))
```

Comment:
Here is another breaking change. `string emailAddress` was rearrange, please move it back to the end by modifying the order in the swagger file.

Reply:
Hi @JonathanCrd, the Email field is at the end only in swagger - 
https://github.com/Azure/azure-rest-api-specs/blob/f69f2e8053ac8ae352b11c33655234dea4cf4cfd/specification/confluent/resource-manager/Microsoft.Confluent/preview/2021-03-01-preview/confluent.json#L813